e2e/api/steps.js

```diff
@@ -57,8 +57,15 @@ module.exports = {
 
     await assertCorrectEventsAreAvailableToUser(config.solicitorUser, 'AWAITING_CASE_NOTIFICATION');
     await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'AWAITING_CASE_NOTIFICATION');
+    let i;
+    for(i=0; i<data[eventName].invalid.Court.courtLocation.applicantPreferredCourt.length; i++) {
+      await assertError('Court', data[eventName].invalid.Court.courtLocation.applicantPreferredCourt[i],
```

Comment:
Can we please have `null` as 3rd argument here, instead of `'The data entered is not valid for this type of field.'`?

It's currently a bit misleading because we actually don't assert it in `assertError` method.

Reply:
all done!